README.md

```diff
@@ -12,23 +12,22 @@ This is a sample repository showcasing how to integrate the Native Checkout expe
     - [Application Creation](#application-creation)
     - [NVM](#nvm)
     - [Docker](#docker)
-    - [Carthage](#carthage)
 3. [Setup](#setup)
-4. [Additional Info](#additional-info)
-5. [Troubleshooting](#troubleshooting)
+    - [Set Client ID / Client Secret / Return URL](#1-set-client-id--client-secret--return-url)
+    - [Setup Dependencies](#2-setup-dependencies)
+4. [Troubleshooting](#troubleshooting)
     - [NVM Issues](#nvm-issues)
-    - [Carthage Issues](#carthage-issues)
 <!-- /TOC -->
 
 ## Quick Setup
 
 ### Create Application ID
 
-Create an application at [PayPal Applications](https://developer.paypal.com/developer/applications/). You can retrieve your client id and secret there.
+Create an application at [PayPal Applications](https://developer.paypal.com/developer/applications/). You can retrieve your client id, client secret, and create a return url there.
 
 ### Application Scopes
 
-In order to set the proper application scope permissions follow the documentation for settings scopes at [Initial Setup Documentation](https://developer.paypal.com/docs/limited-release/native-checkout/setup/#).
+In order to set the proper application scope permissions follow the documentation for settings scopes at [Initial Setup Documentation](https://developer.paypal.com/docs/limited-release/native-checkout/setup/#opt-in-to-native-checkout).
```

Comment:
I wonder if we can get the URL for the non-limited release docs and just wait to merge this PR until soft-launch date? That way we don't have to immediately update these.

Reply:
Good call!!!
Yep I just asked Jon, for now I think we can keep this, and Jon will let us know when we have new URLs for the new docs!!!